src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,6 +975,29 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
+    /// <summary>
+    /// Obsolete class of ShowHidePreviewBubblesConverter
+    /// Using of this class will produce compile warnings
+    /// TODO: To be removed in Dynamo 2.0
+    /// </summary>
+    [System.Obsolete("This class is obsolete, string converter for Preview Bubble Display Option only returns single value now. ")]
+    public class ShowHidePreviewBubblesConverter : IValueConverter
+    {
+        #region IValueConverter Members
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return Resources.DynamoViewSettingsMenuShowPreviewBubbles;
```

Comment:
but won't that cause a crash??

Imagine someone installs Revit 1.1 or 1.0 and has core 1.2 somehow, it **IS** possible for them to get into this state.

Let's just argue for a minute that someone like Revit which has a reference to DynamoCoreWPF - calls this public converter for some reason ... it'll crash right? ... what does that resource point to now...?